docker/test/integration/minifi/core/ImageStore.py

```diff
@@ -23,21 +24,28 @@ def cleanup(self):
             self.client.images.remove(image.id, force=True)
 
     def get_image(self, container_engine):
+        if container_engine in self.images:
```

Comment:
Good improvement. With this change I think we should also rename the `__get_.*_image` functions to `__build_.*image`